dspace/janitor.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "DSpace",
+  "description": "DSpace is a turnkey institutional repository application.",
+  "icon": "https://janitor.technology/img/dspace.svg",
+  "docker": {
+    "image": "janitortechnology/dspace"
+  },
+  "ports": {
+    "22": {
+      "label": "SSH",
+      "proxy": "none"
+    },
+    "5432": {
+      "label": "DB",
+      "proxy": "https"
+    },
+    "8080": {
+      "label": "VNC",
+      "proxy": "https",
+      "preview": true
+    },
+    "8089": {
+      "label": "Cloud9",
+      "proxy": "https",
+    }
+  },
+  "scripts": {
+    "Make an administrator": {
+      "cwd": "/dspace/bin",
+      "cmd": "sudo ./dspace create-administrator"
+    },
+    "Run DSpace": {
+      "cmd": [
+        "sudo service postgresql start",
+        "sudo service tomcat8 start"
```

Comment:
Wow, I didn't know this actually worked (making `cmd` an array). Where did you find out about this? It will probably break our [Janitor Cloud9 plugin](https://github.com/JanitorTechnology/c9.ide.janitorconfig/blob/0698b58d841e6c4948040933b9fdf088b3f21ccd/c9.ide.janitorconfig.js#L68) because it currently expects `cmd` to always be a String.

Reply:
That originates from the C9 documentation: https://docs.c9.io/docs/custom-runners

Reply:
Thanks a lot for finding this! I filed this issue to support it in our plugin: https://github.com/JanitorTechnology/c9.ide.janitorconfig/issues/14